packages/cody/src/lib/hooks/on-document.ts

```diff
@@ -102,6 +102,15 @@ export const onDocument: CodyHook<Context> = async (vo: Node, ctx: Context) => {
 
       withErrorCheck(registerQueryResolver, [service, vo, ctx, tree]);
 
+      generateFiles(tree, __dirname + '/query-files/fe', ctx.feSrc, {
+        tmpl: "",
+        service: serviceNames.fileName,
+        serviceNames,
+        voNames,
+        ns,
+        ...queryNames
```

Comment:
For the main target of the code generation operation (in this case the query) the `names()` object is directly merged into the substitutions object. This is more a less convention and has the advantage that you can use for example `__fileName__.ts` in a template filename and it gets replaced with the query file name. For file and folder name replacements you can only use string substitutions and not nested objects like in the EJS templates.